Release influxctl v2.9.5 and v2.9.6 (#5565)
influxdata/docs-v2: content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/login.md

---
title: influxctl auth login
description: >

>   The `influxctl auth login` command logins with InfluxData Auth0 or InfluxDB

Reviewer: ```suggestion
  The `influxctl auth login` command lets a user log in to an InfluxDB cluster using InfluxData Auth0 or an InfluxDB
```

  Clustered identity provider
menu:

>   influxdb_clustered:

Reviewer: ```suggestion
  influxdb_cloud_dedicated:
```

>   {{< product-name omit="Clustered" >}} identity provider

Reviewer 2: Unfortunately, Hugo shortcodes don't work in page descriptions.

```suggestion
  the InfluxDB Cloud Dedicated identity provider.
```

    parent: influxctl auth
weight: 301
---

The `influxctl auth login` command logins with InfluxData Auth0 or InfluxDB

> Clustered identity provider

Reviewer: > Clustered

Does this also apply to Cloud Ded? If so, we can use the `{{< product-name omit="Clustered" >}}` short tag. If not, I can remove it here and keep it in `/clustered...`.

> The `influxctl auth login` command logins with InfluxData Auth0 or InfluxDB
> Clustered identity provider

Reviewer: ```suggestion
The `influxctl auth login` command lets a user log in to an InfluxDB cluster using InfluxData Auth0 or an InfluxDB Clustered identity provider.
```

> The `influxctl auth login` command lets a user log in to an {{< product-name >}}

Reviewer: ```suggestion
The `influxctl auth login` command lets a user log in to an {{< product-name omit="Clustered" >}}
```
